xml/ha_config_cli.xml

```diff
@@ -191,22 +194,34 @@
        agents).
       </para>
      </formalpara>
+      <para>It is possible to abbreviate subcommands as long as they are
+        unique. For example, you can shorten <command>status</command> as
+      <command>st</command> and &crmsh; will know what you have meant.
+      </para>
+      <para>Another feature is to shorten parameters. Usually, you add
+        parameters through the <command>params</command> keyword.
+        You can leave out the params section if it the first and only section.
+        For example, this line:
+      </para>
+      <screen>&prompt.root;<command>crm</command> primitive ipaddr ocf:heartbeat:IPaddr2 params ip=192.168.0.55</screen>
+      <para>is equivalent to this line:</para>
+      <screen>&prompt.root;<command>crm</command> primitive ipaddr ocf:heartbeat:IPaddr2 ip=192.168.0.55</screen>
     </listitem>
     <listitem>
      <formalpara>
-      <title>As crm Shell Script</title>
+      <title>As crm Shell Script:</title>
       <para>
        Use <command>crm</command> and its subcommands in a script. This can
        be done in two ways:
       </para>
      </formalpara>
-<screen>&prompt.root;<command>crm</command> -f script.cli
-&prompt.root;<command>crm</command> &lt; script.cli</screen>
+<screen>&prompt.root;<command>crm</command> -f example.cli
+&prompt.root;<command>crm</command> &lt; example.cli</screen>
      <para>
       The script can contain any command from <command>crm</command>. For
```

Comment:
Hmn... actually I wanted to say is that the script can contain **any subcommand** of crm. Is that better?